### ISSUE TITLE
Adding new material

### ISSUE THREAD

**Issue by DS-Sagar** [NREL/bifacial_radiance#302]
There is number of materials available by default in list. I want to add new one can any one help me to do the same. 

I want to know what each and every number means and how it affects the reflection property of that material

#G173 light soil .21R
void plastic litesoil
0
0
5 .29 .187 .163 0 0

**Comment by DS-Sagar** [comment]
Thank you for clarification @nicorie. I want to add mirror type material which is having more reflective property

**Comment by jalderman9** [comment]
@DS-Sagar I've tinkered with some mirror materials.  Here's what I used.

First, I defined a glass material with an index of refraction of 1.25:
```
void glass module_glass
0
0
4 0.9831 0.9831 0.9831 1.25 
```

Then I defined a mirror with 80% reflectance:
```
void mirror glass_mirror
1 module_glass
0
3 0.08 0.08 0.08
```

Then make your object and specify the glass_mirror material.

There's more info on the mirror material on page 16 of the user's manual: https://floyd.lbl.gov/radiance/refer/usman2.pdf

Note that if you just model glass without the mirror material (module_glass), you'll also get reflections in case that's what you're after:
![mississippi2](https://user-images.githubusercontent.com/31673048/110554474-b9f03b80-80ef-11eb-9c0d-8766c91a0b97.png)

**Comment by DS-Sagar** [comment]
Thank you @jalderman9

**Comment by DS-Sagar** [comment]
@jalderman9 have got this image by converting .oct into the some image file or it is some other file. If it is converted from .oct can you please tell me the procedure to convert .oct to any image format. Thank You!!!

**Comment by jalderman9** [comment]
@DS-Sagar To render an oct file as an image I use [accelerad](https://nljones.github.io/Accelerad/) which is a GPU accelerated version of the radiance `rpict` rendering tool.

There are several steps:
1) Render the oct file as a radiance hdr image, passing in the .vf view file (viewable with `ximage`)
2) Perform scaling and anti-aliasing with `pfilt`
3) Convert the image to tif format
4) Convert the tif to png or jpg

I have a shell script that does:
`ACCEL="-x 11000 -y 6200 -ss 8 -st 0.004 -ac 65535 -at 0.01 -an 180 -aa 0 -ab 5 -ad 880 -ar 450 -as 280"
PFILT="-x /5 -y /5 -e -0.66"
export RAYPATH=/usr/local/radiance/lib:/usr/local/accelerad/lib:./ ; 
export LD_LIBRARY_PATH=/usr/local/accelerad/bin ; 
export PATH=/usr/local/accelerad/bin:/usr/local/radiance/bin:$PATH ; 
accelerad_rpict $ACCEL -vf myviewfile.vf myoctfile.oct > myrender.hdr &&
 pfilt $PFILT myrender.hdr > myrender.pfilt &&
 ra_tiff myrender.pfilt myrender.tif &&
 convert myrender.tif myrender.png &&
 rm myrender.tif `
 
 That creates a 2200 pixel wide rendering.  On an aws g4dn.12xlarge GPU instance that process takes about 5 minutes.

**Comment by DS-Sagar** [comment]
Hello @jalderman9 I am defining the material using same technique as you used for defining the glass material in ground.rad file. But program is giving error as "undefined modifier" can you please help me in the same

**Comment by shirubana** [comment]
fyi -- not sure if this is in the currently released branch or on development, but 

``makeModule`` has a glass feature that can be added, with the default glass material defined in ground rad.
``addMaterial`` function can help add materials to the ground.rad programatically, so giving the input values there might be useful to avoid errors on writing

**Comment by cdeline** [comment]
Fixed in release 0.4.0
